elements/pfelement/src/pfelement.js

```diff
@@ -591,7 +595,8 @@ class PFElement extends HTMLElement {
     for (let mutation of mutationsList) {
       // If a new node is added, attempt to cascade attributes to it
       if (mutation.type === "childList" && mutation.addedNodes.length) {
-        this.cascadeProperties(mutation.addedNodes);
+        const nonTextNodes = [...mutation.addedNodes].filter(n => n.nodeType !== HTMLElement.TEXT_NODE);
```

Comment:
Nice!

Reply:
Maybe the non-text node could be checked for inside the cascadeProperties instead/as well?  I'm imagining components using this and not having the cleansing set up there.

Reply:
cascadeProperties already has text node filtering.  Snip:
```js
              // if this node has a match function (i.e., it's an HTMLElement, not
              // a text node), see if it matches the selector, otherwise drop it (like it's hot).
              if (nodeItem.matches && nodeItem.matches(selector)) {
```